node/parachain/src/command.rs

```diff
@@ -268,12 +269,55 @@ pub fn run() -> Result<()> {
 		None => {
 			let runner = cli.create_runner(&*cli.run)?;
 			let collator = cli.run.base.validator || cli.collator;
-			let author_id: Option<H160> = cli.run.author_id;
-			if collator {
-				if author_id.is_none() {
-					return Err("Collator nodes must specify an author account id".into());
-				}
+
+			// Supply the correct author id for wellknown validators.
+			// This isn't super elegant, but the alternative is modifying Substrate
+			// and this will go away when we start signing blocks
+			let author_id = if cli.run.base.shared_params.dev {
+				//TODO what do we actually expect `--dev` to do in the parachain context
```

Comment:
> TODO what do we actually expect `--dev` to do in the parachain context

Idk, could combine dev and alice branches if we're using the alice keys for both

Reply:
I believe the `--dev` will serve for the blockchain (standalone) only

Reply:
I've combined then as @4meta5  suggested in the standalone, and made the parachain fail with a clear error message that `--dev` is not supported. (Of course that will change again when #204 lands).